webpack/assets/javascripts/react_app/common/I18n.js

```diff
@@ -16,9 +16,7 @@ class IntlLoader {
     await this.fetchIntl();
     addLocaleData(
       await import(
-        /* webpackChunkName: 'react-intl/locale/[request]' */ `react-intl/locale-data/${
-          this.locale
-        }`
+        /* webpackChunkName: 'react-intl/locale/[request]' */ `react-intl/locale-data/${this.locale}`
```

Comment:
Any idea if the comment would make its job with a `\n` newline separating it from the string?